app/views/home/index.html.erb

```diff
@@ -1,47 +1,70 @@
+
 <div class="parallax">
   <div id="section-1" class="parallax-section">
     <div class="parallax-layer parallax-layer-base">
-      <div class="full-bg-img flex-center">
+      <div class="full-bg-img flex-center ">
         <h1 class="h1-responsive log"><a href="/games/new"><button id="play-button" type="button" name="Play Now"> Play Now</button></a></h1>
       </div>
     </div>
-    <div class="parallax-layer parallax-layer-back">
-    </div>
-  </div>
-  <div class="container">
-    <div id="game-video">
-    <img src="/game-gif.gif" alt="Loading" title="Loading" />
-    </div>
   </div>
-  <div class="container">
-  <div class="iconcontainer">
-    <div class="row">
-      <div class="col-xs-12 col-sm-6 col-md-6 col-lg-6">
-        <div class="iconbox">
-          <div class="iconbox-icon">
-            <span class="glyphicon glyphicon-book"></span>
-          </div>
-          <div class="featureinfo about-game">
-            <h4 class="text-center h1-responsive">About This Game:</h4>
-            <p>Lava Typer is a fun game to help people improve their typing speed and accuracy.</p>
+  <div class="container-rules">
+        <div class="features-list">
+            <div data-100-bottom-top="transform: translateX(-200px); opacity: 0" data-center-top="transform: translateX(0px); opacity: 1">
+              <i class="material-icons">flag</i>
```

Comment:
Please be careful with indentation, it can get a little confusing especially when there are many nested elements. It is a little hard for me to read.